server/client.go

```diff
@@ -932,6 +1045,11 @@ func (c *client) processMsg(msg []byte) {
 	}
 }
 
+func (c *client) pubPermissionViolation(subject []byte) {
+	c.sendErr(fmt.Sprintf("Permissions Violation for Publish to %q", subject))
+	c.Errorf("Publish Violation - User %q, Subject %q", c.opts.Username, subject)
```

Comment:
I was going back and forth between if this should be an error or a debug statement. When you try to connect to an authorized server its an error, so we kinda already have this issue in a different form already. Thoughts?